src/Controller/ActionHandler.py

```diff
@@ -168,45 +183,55 @@ def init_windowing_menu(self):
         def generate_triggered_handler(text_):
             def handler(state):
                 self.windowing_handler(state, text_)
+
             return handler
+
         windowing_actions = []
         for name in names_ordered:
             text = str(name)
             action_windowing_item = QtGui.QAction(self.menu_windowing)
-            action_windowing_item.triggered.connect(generate_triggered_handler(text))
+            action_windowing_item.triggered.connect(
+                generate_triggered_handler(text))
             action_windowing_item.setText(text)
             windowing_actions.append(action_windowing_item)
 
-        # For reasons beyond me, the actions have to be set as a child of the windowing menu *and* later be added to
-        # the menu as well. You can't do one or the other, otherwise the menu won't populate.
-        # Feel free to try fix (or at least explain why the action has to be set as the windowing menu's child twice)
+        # For reasons beyond me, the actions have to be set as a child of
+        # the windowing menu *and* later be added to the menu as well. You
+        # can't do one or the other, otherwise the menu won't populate. Feel
+        # free to try fix (or at least explain why the action has to be set
+        # as the windowing menu's child twice)
         for item in windowing_actions:
             self.menu_windowing.addAction(item)
 
     def save_struct_handler(self):
         """
-        If there are changes to the RTSTRUCT detected, save the changes to disk.
+        If there are changes to the RTSTRUCT detected,
+        save the changes to disk.
         """
         if self.patient_dict_container.get("rtss_modified"):
             self.__main_page.structures_tab.save_new_rtss()
         else:
-            QtWidgets.QMessageBox.information(self.__main_page, "File not saved",
-                                              "No changes to the RTSTRUCT file detected.")
+            QtWidgets.QMessageBox.information(
+                self.__main_page, "File not saved",
+                "No changes to the RTSTRUCT file detected.")
 
     def windowing_handler(self, state, text):
         """
         Function triggered when a window is selected from the menu.
-        :param state: Variable not used. Present to be able to use a lambda function.
+        :param state: Variable not used. Present to be able to use a lambda
+        function.
         :param text: The name of the window selected.
         """
         # Get the values for window and level from the dict
-        windowing_limits = self.patient_dict_container.get("dict_windowing")[text]
+        windowing_limits = self.patient_dict_container.get("dict_windowing")[
```

Comment:
i think breaking the line there doesn't make sense. how about breaking the line at `.get("dict_windowing")[text]`?

Reply:
Thats a good idea, PEP-8 prefers splitting the line at the bracket, but '\' notation is acceptable so I'll do that